org.eclipse.cpacep/test/org/eclipse/cpacep/util/CPACEPConnectorTest.java

```diff
@@ -0,0 +1,72 @@
+package org.eclipse.cpacep.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CPACEPConnectorTest {
+	String home = System.getProperty("user.home");
+	String tempDirectory = System.getProperty("java.io.tmpdir");
+	//String homePath = "/home/supun/Documents/cpachecker";
+	//Path homePath = Paths.get(File.separator + "home").resolve("supun").resolve("Documents").resolve("CPAchecker-1.7-unix");
+	//Path homePath = Paths.get(home, "Documents", "CPAchecker-1.7-unix");
+	Path homePath = Paths.get(home, "build", "SupunArunoda", "CPACEP","cpachecker");
+	//Path homePath = StringHandler.getHomePath(File.separator + "home" + File.separator + "travis" + File.separator + "build" + File.separator + "SupunArunoda" + File.separator + "cpachecker");
+	StringBuilder sb;
+	String lcSpecification;
+	String lcConfiguration;
+	String result;
+
+	Path outputPathOriginal = Paths.get(tempDirectory, "original");
+	File outputDirectory;
+
+	@Before
+	public void setup() {
+		CPACEPConnector cpacepConnector = new CPACEPConnector();
+		lcSpecification = "default";
+		lcConfiguration = "default";
+		try {
+			outputDirectory = FileHandler.createTempDirectory();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			throw new AssertionError();
```

Comment:
How about you include the original exception in the AssertionError (`new AssertionError(e)`) so that users can see the IOException that occurred?